dtfs-central-api/src/v1/controllers/portal/facility/delete-amendment.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { ApiError, AUDIT_USER_TYPES, CustomExpressRequest } from '@ukef/dtfs2-common';
+import { HttpStatusCode } from 'axios';
+import { Response } from 'express';
+import { validateAuditDetailsAndUserType } from '@ukef/dtfs2-common/change-stream';
+import { PortalFacilityAmendmentService } from '../../../../services/portal/facility-amendment.service';
+import { DeletePortalFacilityAmendmentPayload } from '../../../routes/middleware/payload-validation/validate-delete-portal-facility-amendment-payload';
+
+type DeletePortalAmendmentRequestParams = { facilityId: string; amendmentId: string };
+export type DeletePortalAmendmentRequest = CustomExpressRequest<{ params: DeletePortalAmendmentRequestParams; reqBody: DeletePortalFacilityAmendmentPayload }>;
+
+/**
+ * delete portal facility amendment
+ * @param req - request
+ * @param res - response
+ */
+export const deletePortalAmendment = async (req: DeletePortalAmendmentRequest, res: Response) => {
+  const { facilityId, amendmentId } = req.params;
+  const { auditDetails } = req.body;
+
+  try {
+    validateAuditDetailsAndUserType(auditDetails, AUDIT_USER_TYPES.PORTAL);
```

Comment:
Can we please validate facilityId and amendmentId are valid, there are existing functions which can be used to validate ObjectID syntax.